web/src/components/storage/LvmPage.tsx

```diff
@@ -132,7 +133,7 @@ export default function LvmPage() {
       <Page.Content>
         <Form id="lvmForm" onSubmit={onSubmit}>
           {errors.length > 0 && (
-            <Alert variant="warning" isInline title={_("Something went wrong")}>
+            <Alert variant="warning" isInline title={_("Check the following before continuing")}>
```

Comment:
NP: Should it end with colons?

Reply:
I don't think so. It's acting as a title.

> Avoid using full stops in titles and headings unless they are complete sentences. https://freelingual.com/full-stop-punctuation

Also having a look to the [SUSE documentation style guide](https://documentation.suse.com/style/current/html/docu_styleguide/art-styleguide.html) I've noticed they do not use it in titles neither